docs/plan/PR2_GLOBAL_STATE_PERSIST_0128.md

```
@@ -0,0 +1,1099 @@
+// 변경 전
+this.scene.load.image(mapKey, `/assets/maps/desert_stage${stageNum}.webp`);
+
+// 변경 후
+this.scene.load.image(mapKey, `${API_URL}/api/maps/${index}`);
+```
+
+---
+
+## 작업 순서
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)